### PR TITLE
fix: add /usr/sd-static/bin to PATH

### PR DESCRIPTION
**Pull request by DekusDenial**
provide bin path to statically linked executables

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by klu909 — APPROVED** [review]

File: executor/executor.go [hunk 1]
```diff
@@ -157,7 +157,7 @@ func doRunCommand(guid, path string, emitter screwdriver.Emitter, f *os.File, fR
 // Executes teardown commands
 func doRunTeardownCommand(cmd screwdriver.CommandDef, emitter screwdriver.Emitter, shellBin, exportFile, sourceDir string, stepExitCode int) (int, error) {
 	shargs := []string{"-e", "-c"}
-	cmdStr := "export PATH=${PATH}:/opt/sd:/usr/sd/bin SD_STEP_EXIT_CODE=" + strconv.Itoa(stepExitCode) + " && " +
+	cmdStr := "export PATH=${PATH}:/opt/sd:/usr/sd/bin:/usr/sd-static/bin SD_STEP_EXIT_CODE=" + strconv.Itoa(stepExitCode) + " && " +
```
Comment (jithine) [inline executor/executor.go:160]: why not reuse existing `/usr/sd/bin` ?

Comment (DekusDenial) [reply executor/executor.go:160]: like what the name implies, need to separate static and dynamic binaries

**Comment by adong** [comment]
I was told the use case is to use statically compiled binaries
@kumada626

**Comment by DekusDenial** [comment]
like git, as we tried to move away from using habitat

**Pull request merged**